internal/storage/fs/snapshot_test.go

```diff
@@ -162,11 +164,26 @@ func (fis *FSIndexSuite) TestListFlags() {
 			name:      "Sandbox",
 			namespace: "sandbox",
 		},
+		{
+			name:      "Production",
+			namespace: "production",
+			pageToken: "foo",
+			listError: errors.New("pageToken is not valid: \"foo\""),
+		},
+		{
+			name:      "Production",
+			namespace: "production",
+			pageToken: "60000",
+			listError: errors.New("invalid offset: 60000"),
+		},
 	}
 	for _, tc := range testCases {
 		t.Run(tc.name, func(t *testing.T) {
-			flags, err := fis.store.ListFlags(context.TODO(), tc.namespace, storage.WithLimit(5), storage.WithPageToken("0"))
-			assert.NoError(t, err)
+			flags, err := fis.store.ListFlags(context.TODO(), tc.namespace, storage.WithLimit(5), storage.WithPageToken(tc.pageToken))
+			if err != nil {
+				assert.EqualError(t, err, tc.listError.Error())
+				return
+			}
```

Comment:
This is just a nicer experience for when the case unexpectedly returns a non-nil error.
Otherwise, if `listError` is expected to be nil, but list returns a non-nil error, then this original line would panic on `tc.listError.Error()` with a nil pointer dereference.
```suggestion
			if tc.listError != nil {
				assert.EqualError(t, err, tc.listError.Error())
				return
			}
			require.NoError(t, err)
```